src/mgr/ActivePyModules.cc

```diff
@@ -14,25 +14,26 @@
 // Include this first to get python headers earlier
 #include "Gil.h"
 
+#include "ActivePyModules.h"
+
+#include <rocksdb/version.h>
```

Comment:
The make check failure is related https://jenkins.ceph.com/job/ceph-pull-requests/86369/consoleFull#-911630402a811ea2-3e7b-466b-84b4-d13df7e35809

Reply:
See this comment: https://github.com/ceph/ceph/pull/43593#discussion_r738511826

Basically, `#include "rocksdb/include/rocksdb/version.h"` passes make check, but it was requested that I change the include to `#include <rocksdb/version.h>`.